.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 /.github/  @youwol/continuous-integration
+/.github/  @youwol/dev-front
```

Comment:
Only team @youwol/continuous-integration should have ownership of `/.github/`
Give team @youwol/dev-front ownership to _other_ part of the code (see [Github Documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax))

Reply:
i will revert it as before